### ISSUE TITLE
.webp format not supported anymore

### ISSUE THREAD

**Issue by Arno989** [lextm/windowsterminal-shell#15]
In my last version of the script (about 11 days ago or something) I used a .webp image for my python icon and it worked flawlessly, but now I get an error 
![image](https://user-images.githubusercontent.com/32594687/82755782-34a4d000-9dd6-11ea-82e1-1559d20c2a9c.png)
This is probably a handling in icon images you've added but since I haven had any problems with random formats and sizes I don't understand the change.

**Comment by lextm** [comment]
Windows Terminal supports a variety of image formats, but Windows Explorer context menu only supports .ico.

In order to keep the script simple, it is intentional to only support .ico files, but .png files are also supported (because we need to read image cache from Windows Terminal's folders which are in .png format). That's why there was a warning.

I will try to revise the script to eliminate the errors though.

Workaround: You can always use another tool to convert .webp to .png or .ico.

**Comment by Arno989** [comment]
Yeah I thought of the workaround as last resort because I find it annoying if I have to go and convert images to icons and stuff, always takes way longer that it's supposed to and I hate doing it.

I was extremely pleasantly surprised when the .webp just worked out of the box so it was quite a bummer to then see that it didn't anymore. But thanks for looking into it anyway.

**Comment by lextm** [comment]
Sounds like it might be possible to use Windows 10's builtin API, https://github.com/DavidAnson/ConvertTo-Jpeg/blob/master/ConvertTo-Jpeg.ps1

Will leave this issue open for a few days to see if that API can be reused for .webp to .ico conversion.
